websites/K/Kosmi/presence.ts

```diff
@@ -0,0 +1,44 @@
+const presence = new Presence({
+    clientId: "813518808634621952"
+}),
+
+ browsingStamp =  Math.floor(Date.now() / 1000);
+
+let details: string;
+
+presence.on("UpdateData", async () => {
+
+      switch (location.host) {
+        case "kosmi.io":
+            details = "On the Startpage";
+            break;
+        case "app.kosmi.io": 
+            if (location.pathname === "/") {
+                details =  "Viewing the Home Page...";
+            }
+                
+            else if (location.pathname === "/lobby") {
+                details = "Browsing public rooms...";
+            }
+                
+            else if (location.pathname.includes("/room/")) {
+                details = "In a room";
+            }
+                    
+            break;
+      default: break;
+    }
```

Comment:
Currently a low quality presence, add some more statues or some interactive information (Like reading something of an element to display in status)